viewer/src/datamodels/cad/InstancedMeshManager.test.ts

```diff
@@ -0,0 +1,109 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { Matrix4 } from 'three';
```

Comment:
Very minor but I think we should use `THREE.Matrix4`

Reply:
yup, import mistake. Ill fix